<launch>
    <node pkg="hrl_clickable_display" name="clickable_display" type="clickable_display">
        <remap from="image" to="/arrow_overlaid" />
        <remap from="l_mouse_click" to="l_mouse_click" />
        <remap from="r_mouse_click" to="r_mouse_click" />
    </node>
</launch>
